src/app/portlets/dot-edit-page/shared/services/dot-edit-layout.service.ts

```diff
@@ -105,7 +109,10 @@ export class DotEditLayoutService {
                             .map(
                                 (dotContainersColumnBox: DotContainerColumnBox) =>
                                     <DotPageContainer>{
-                                        identifier: dotContainersColumnBox.container.identifier,
+                                        identifier:
```

Comment:
I am not reaaly sure if all this change is test into the spec file, I not see any expect for any path for example.

Reply:
this, https://github.com/dotCMS/core-web/pull/938/files#diff-9fdb7f7a5d8c1c883513d66fe700c574R244, evaluate the change